app/services/mapping/declaration/DeclarationBuilder.scala

```diff
@@ -33,7 +33,7 @@ object DeclarationBuilder {
     declaration.setTypeCode(TypeCodeBuilder.build)
     declaration.setGoodsItemQuantity(GoodsItemQuantityBuilder.build)
     declaration.setAgent(AgentBuilder.build)
-    declaration.setGoodsShipment(GoodsShipmentBuilder.build)
+//    declaration.setGoodsShipment(GoodsShipmentBuilder.build)
```

Comment:
I think this whole class will eventually be redundant.   Commenting out calls until I'm certain of this.

Reply:
Yeh this class can go, maybe see how much it breaks if you delete it

Reply:
I was going for a "managed decline" to try and keep the number of changes in a PR down